struts2-jquery-plugin/src/main/resources/template/jquery/head.ftl

```diff
@@ -23,6 +23,8 @@
     <#assign jQueryVersion="2.2.4">
 </#if>
 <#assign jQueryUIVersion="1.12.1">
+<#-- issue89: temporary fix because of i18n files not being available for current jQuery UI version -->

```

Comment:
@jogep
Personaly i don't have a clue, this was the only fix i could think of for now.

I didn't see any message by them saying where they're located either